incubator/hnc/internal/kubectl/configset.go

```diff
@@ -37,12 +38,21 @@ var setTypeCmd = &cobra.Command{
 		flags := cmd.Flags()
 		apiVersion, _ := flags.GetString("apiVersion")
 		kind, _ := flags.GetString("kind")
+		force, _ := flags.GetBool("force")
 		config := client.getHNCConfig()
 
 		exist := false
 		for i := 0; i < len(config.Spec.Types); i++ {
 			t := &config.Spec.Types[i]
 			if t.APIVersion == apiVersion && t.Kind == kind {
+				if t.Mode == api.Ignore && mode == api.Propagate && !force {
+					fmt.Println("Switching directly from 'Ignore' to 'Propagate' mode might cause source overwriting conflict.")
```

Comment:
Last change - users won't know what "source overwriting conflict" means. Let's use more natural language: "could cause existing %s objects in child namespaces to be overwritten by objects from ancestor namespaces" (where `%s` is `t.Kind`). The rest of the message lgtm.

Reply:
done